### DISCUSSION TITLE
Shutdown process

### DISCUSSION THREAD

**Discussion by elzinko** [Drakkar-Software/OctoBot#2204] [Q&A] [answered]
Hello,
Does a shutdown process already exists ? 
I cannot find one in the documentation. I mean I would like to redeploy my octobot that is locally hosted with the assurance that all positions are closed. I mean that if I close all manually, some order could be opened in the meantime.
Thanks for help.

**Comment by jimoquinn** [comment, marked as answer]
I think you best bet is to check at the exchange for both open orders and open positions.  You could check before you shutdown octobot and then afterwards, which may provide some insight as to octobot's behavior 

I seem to recall that when I stopped one of the DCA strats that I was running on the cloud octobot, I had to manually close a bunch of open orders on the exchange (the DCA strat never sells at a loss, which could be a factor?).
